src/DssSpell.t.sol

```diff
@@ -839,22 +844,32 @@ contract DssSpellTest is DssSpellTestBase {
 
     // SPELL-SPECIFIC TESTS GO BELOW
 
-    string RWA009_OLDDOC = "QmfEgZuiw6wsTRUYerdPZNUrqDXSGM6Nm4fM3nG7nNbEjT";
-    string RWA009_NEWDOC = "QmPzuLuJ5Xq6k6Hbop1W5s4V9ksvafYoqcW9sU5QRwz5h1";
+    function testDIRECTSPARKDAIBuffer() public {
+        DirectSparkDaiPlanLike DIRECT_SPARK_DAI_PLAN = DirectSparkDaiPlanLike(addr.addr("DIRECT_SPARK_DAI_PLAN"));
 
-    function testRWA009DocChange() public {
-        _checkRWADocUpdate("RWA009-A", RWA009_OLDDOC, RWA009_NEWDOC);
+        assertEq(DIRECT_SPARK_DAI_PLAN.buffer(), 30 * MILLION * WAD);
+
+        _vote(address(spell));
+        _scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done(), "TestError/spell-not-done");
+
+        assertEq(DIRECT_SPARK_DAI_PLAN.buffer(), 50 * MILLION * WAD);
     }
 
-    function testPushGUSDOutInputConduit() public {
+    function testPushPAXOutInputConduit() public {
+        DSTokenAbstract gem = DSTokenAbstract(addr.addr("PAX"));
+        address MCD_PSM_PAX_A_INPUT_CONDUIT_JAR = addr.addr("MCD_PSM_PAX_A_INPUT_CONDUIT_JAR");
         uint256 prevDai = vat.dai(address(vow));
-        DSTokenAbstract gem = DSTokenAbstract(addr.addr("GUSD"));
+        uint256 gemBalanceToPush = 754_598_72; // 2 decimal places
```

Comment:
```suggestion
        uint256 gemBalanceToPush = 754_598.72 ether; // `ether` is only a keyword
```
This way you can avoid the ` / 100` below. As for `vat.dai`, you can multiply it by `RAY` instead of `RAD`.